src/views/previous/results.html

```diff
@@ -5,7 +5,7 @@
 {% set hmpoPageKey = "address-previous" %}
 {% set hmpoPageContent = "address-previous" %}
 
-{% block content %}
```

Comment:
content is the entire block. By overriding this we remove the option for an error summary to be shown.